src/doc/src/reference/workspaces.md

```diff
@@ -39,6 +39,9 @@ To create a workspace, you add the `[workspace]` table to a `Cargo.toml`:
 
 At minimum, a workspace has to have a member, either with a root package or as
 a virtual manifest.
+It's also a good practice to specify the `resolver = "2"` unless it is necessary
+to rely on the old one. It is a default resolver for all packages in the `2021` edition
+but it has to be explicitely set under the workspace.
```

Comment:
> I'm not sure if it makes changes to the documentation unneeded. Anyone who just goes through the Workspaces section and tries to use the examples will now face the warning right after

I agree that we should have some workspace reocommendations.  Its been something we've been talking about.  [This zulip thread](https://rust-lang.zulipchat.com/#narrow/stream/246057-t-cargo/topic/Version.20Requirements.20documentation) talks some about what we might want to cover and how to include it but that is much bigger than this change and I don't think you need to tackle it.

For how we are presenting this specific case, I feel like we are doing so without context.  Personally, I would add a "workspace recommendations" section at the end and have a link to that.

Reply:
> For how we are presenting this specific case, I feel like we are doing so without context. Personally, I would add a "workspace recommendations" section at the end and have a link to that.

As it looks that this is specific to virtual workspaces, I moved the example and explanation under this section, and imo it looks good there after also applying your suggestions from the other comment. The "workspace recommendations" section would be great, but I think it should cover more than just this and then probably be separate PR

Reply:
I agree, the new version works well enough.  Once we resolve the extra entry I noted, this should be good to go